iceoryx_binding_c/test/moduletests/test_service_description.cpp

```diff
@@ -32,13 +32,14 @@ using namespace iox::capro;
 
 TEST(iox_service_description_test, StringSizesAreCorrect)
 {
+    constexpr uint64_t ZERO_TERMINATOR_SIZE = 1U;
     ::testing::Test::RecordProperty("TEST_ID", "f32a6d19-9ff7-4913-a1d8-39d46267b114");
     EXPECT_THAT(sizeof(decltype(std::declval<iox_service_description_t>().serviceString)),
-                Eq(iox::capro::IdString_t().capacity()));
+                Eq(iox::capro::IdString_t().capacity() + ZERO_TERMINATOR_SIZE));
```

Comment:
The null terminator is always included in string literals, see
```c
const char blubb[] = "EvilHuhn";
printf("%lu \n", sizeof(blubb) / sizeof(char) ); // = 9, since the zero terminator is always part of the array.
```

Reply:
For literals sure but `IdString_t::capacity` returns a capacity which excludes the null terminator.